services/llm_request_policy.py

```
@@ -120,13 +120,24 @@ async def apply(self, event: AstrMessageEvent, req: ProviderRequest) -> None:
                     )
                     continue
 
+                if not should_expose:
+                    logger.info(
+                        "[文件管理] 文件 %s 已保存为 %s，但当前轮未暴露文件工具，跳过上传文件提示注入",
```

Ruff (0.15.6)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)